IDEAS/Resources/ReferenceResults/Dymola/IDEAS_Utilities_Psychrometrics_Functions_Examples_saturationPressure.txt

```diff
@@ -0,0 +1,9 @@
+last-generated=2016-03-15
```

Comment:
@cprotopa I'm confused where this comes from? The correct reference result has a capital S in saturationPressure so this seems to be a duplicate. Can you remove it from the pull request?

Reply:
I have no idea where this comes from, maybe one bad manipulation a while ago. It's weird because on my pc I also have this file with a capital S. I'll have to figure out how to remove this commit...

Reply:
Some file systems make a distinction between files with capital and non-capital letters while others do not. That can lead to funny conflicts. I'm not sure whether windows does. In any case, when I have it, I have to remove the file from git in one commit and then add the file again with the correct capitalization in a second commit.